README.md

```diff
@@ -12,7 +12,7 @@ Person B should modify content here.
 
 ## Person C:
 
-Person C should modify content here.
+Person C should modify content here. Hello, it's ga-young!
```

Comment:
I think it would be better to just have the comment instead of also the instruction.
"Hello, it's ga-young!"